src/sasctl/pzmm/write_json_files.py

```diff
@@ -38,25 +43,24 @@ class JSONFiles:
     @classmethod
     def writeVarJSON(cls, inputData, isInput=True, jPath=Path.cwd()):
         """
-        Writes a variable descriptor JSON file for input or output variables,
-        based on an input dataframe containing predictor and prediction columns.
-
-        The following files are generated by this function:
-        * 'inputVar.json' or 'outputVar.json'
-            Output JSON file located at jPath.
+        Writes a variable descriptor JSON file for input or output variables, based on input data containing predictor
+        and prediction columns.
 
         Parameters
-        ---------------
+        ----------
         inputData : dataframe or list of dicts
-            Input dataframe containing the training data set in a
-            pandas.Dataframe format. Columns are used to define predictor and
-            prediction variables (ambiguously named "predict"). Providing a list of dict
-            objects signals that the model files are being created from an MLFlow model.
-        isInput : boolean
-            Boolean to check if generating the input or output variable JSON.
+            Input dataframe containing the training data set in a pandas.Dataframe format. Columns are used to define
+            predictor and prediction variables (ambiguously named "predict"). Providing a list of dict objects signals
+            that the model files are being created from an MLFlow model.
+        isInput : bool
+            Boolean flag to check if generating the input or output variable JSON.
         jPath : string, optional
-            File location for the output JSON file. Default is the current
-            working directory.
+            File location for the output JSON file. Default is the current working directory.
+
+        Yields
```

Comment:
Take out the Yields section.  The function isn't a generator and never uses `yield`.  It doesn't `return` anything either (besides None) so you can't include the JSON file info in the Returns section either.

See [Numpydoc style guide](https://numpydoc.readthedocs.io/en/latest/format.html#yields)

Best place to document that it writes a file is probably the extended summary or possibly notes section.

Reply:
Resolved.